support/tests/test_tealium_helper.swift

```diff
@@ -30,27 +30,27 @@ enum TealiumTestError : Error {
 }
 
 let testStringArrayValue = ["value1", "value2"]
-let testOptionalData = [TealiumTestKey.stringKey: TealiumTestValue.stringValue as AnyObject,
-                        TealiumTestKey.stringArrayKey: testStringArrayValue as AnyObject] as [String : AnyObject]
+var testOptionalData = [TealiumTestKey.stringKey: TealiumTestValue.stringValue,
+                        TealiumTestKey.stringArrayKey: testStringArrayValue] as [String : Any]
 let testTealiumConfig = TealiumConfig(account:TealiumTestValue.account,
```

Comment:
We should, but in the test_tealium_core/TealiumConfigTests: testSetAndGetOptionalData.